demo/api/src/app.module.ts

```diff
@@ -74,10 +74,68 @@ export class AppModule {
                     inject: [BLOCKS_MODULE_TRANSFORMER_DEPENDENCIES],
                 }),
                 AuthModule,
-                ContentScopeModule.forRoot({
-                    canAccessScope(requestScope: ContentScope, user: CurrentUserInterface) {
-                        if (!user.domains) return true; //all domains
-                        return user.domains.includes(requestScope.domain);
+                UserManagementModule.forRoot<USERMANAGEMENT.pageTree | typeof USERMANAGEMENT.userManagement | "news" | "products">({
+                    // To use a real user service, instantiate this module with forRootAsync and inject required services
+                    userService: {
+                        findUsers: async (args: FindUsersArgs) => {
+                            const search = args.search?.toLowerCase();
+                            const users = staticUsers.filter(
+                                (user) => !search || user.name.toLowerCase().includes(search) || user.email.toLowerCase().includes(search),
+                            );
+                            return [users, users.length];
+                        },
+                        getUser: async (id: string) => {
+                            const index = parseInt(id) - 1;
+                            if (staticUsers[index]) return staticUsers[index];
+                            throw new Error("User not found");
+                        },
+                    },
+                    getAvailablePermissions: async () => ({
+                        news: {
+                            name: "News",
+                        },
+                        products: {
+                            name: "Products",
+                        },
+                        pageTree: {
+                            name: "Page Tree",
+                        },
+                        userManagement: {
+                            name: "User Management",
+                        },
+                    }),
+                    getPermissions: async (user) => {
+                        if (user.email.endsWith("@comet-dxp.com")) {
+                            return USERMANAGEMENT.allPermissions;
+                        } else {
+                            return [{ permission: USERMANAGEMENT.pageTree }, { permission: "news" }];
+                        }
+                    },
+                    getAvailableContentScopes: async () => ({
+                        domain: {
+                            label: "Domain",
```

Comment:
The admin still can offer a translation based on the value. We could change the key here from `label` to `defaultLabel` so that if there is no translation in the admin this text is used.